### DISCUSSION TITLE
The project cannot link to the repository

### DISCUSSION THREAD

**Discussion by Wonderer11** [community/community#11883] [Projects and Issues]
The project cannot link to the repository

**Comment by 294m** [comment]
> The project cannot link to the repository

**Comment by azenMatt** [comment]
Hi - would you be able to expand on the problem you are having here?

**Reply by esoikie** [reply]
I 'think' I may understand.  
- I add a project to my repository
- I go to my repository projects then click on the project
- Once in project view there's no clear way to get back to the associated repository

While I understand that a project can be associated to multiple repositories, it may still be worth while having a default, primary or associated repository connection to help create a bilateral navigation flow  `repository <-> project` instead of unilateral `repository -> project`

To create a workaround for myself I added some quick links into the project's readme so that I can at least click the project details icon in the top right and bring up links back to the primary repository as my project is only associated to that one repository anyways.

**Reply by azenMatt** [reply]
👍 that makes sense - we do have some UI updates cooking to make this easier that we should be able to release soon.

**Reply by Patuca** [reply]
Same here. My workaround was to add a "repository" collum on my list views just to be able no navigate but it doesn't help on the board views. But... part of me likes it because it creates a feeling of a more focused view. I pinned important projects on browser and only use this window to the project itself.

**Reply by Wonderer11** [reply]
> I 'think' I may understand.
> 
>     * I add a project to my repository
> 
>     * I go to my repository projects then click on the project
> 
>     * Once in project view there's no clear way to get back to the associated repository
> 
> 
> While I understand that a project can be associated to multiple repositories, it may still be worth while having a default, primary or associated repository connection to help create a bilateral navigation flow `repository <-> project` instead of unilateral `repository -> project`
> 
> To create a workaround for myself I added some quick links into the project's readme so that I can at least click the project details icon in the top right and bring up links back to the primary repository as my project is only associated to that one repository anyways.

That is what I mean

**Reply by Wonderer11** [reply]
> 👍 that makes sense - we do have some UI updates cooking to make this easier that we should be able to release soon.

Will there be a bi-directional linking between the repository and the project?

**Reply by azenMatt** [reply]
> Will there be a bi-directional linking between the repository and the project?

What do you have in mind for a bi-directional link? 

I've heard some requests for syncing permissions, however that isn't something we are looking to do at the moment.

**Reply by esoikie** [reply]
I think for me, I'm thinking something like.... (off the top of my head)
[![alt](https://screenshot.click/28-24-h9buj-mw10m.png)](https://screenshot.click/28-24-h9buj-mw10m.png)

I put 'default' repository as it could go at the top if set, and I've seen requests for default repository elsewhere, something about having new issues created in a 'default repo' which I think other requests have mentioned (such as a default repo for new issues created within the project board)

**Comment by tekprofessional** [comment]
Another idea might be to have one project per repository.  Then having the option of choosing one project per repository, or one global project board, where issues/tasks are linked to the repo. Just a thought
